_tools/release_docs/main.go

```diff
@@ -151,6 +158,27 @@ func rewriteCodeBlocks(fileBytes []byte) []byte {
 	return []byte(strings.Join(output, "\n") + "\n")
 }
 
+var (
+	titleRE   = regexp.MustCompile(`# (.*)`)
```

Comment:
I think it's a decision to make - some docs dont have a "# foo" but do have a "## foo".  Is it wrong to just take the first header?  Probably - ## should be a sub-header.

Wrinkle: an underlined line apprently renders as an H2, but "feels" like a title.  We should weed those out.